acts_as_archival.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rake"
   gem.add_development_dependency "rr"
   gem.add_development_dependency "sqlite3"
+  gem.add_development_dependency "rubocop", "~> 0.47.1"
```

Comment:
That bit in their README is such a copout (pun intended). I've been using rubocop for multiple years now. THey need to get a damn grip and use semantic versioning. \*flips table\*